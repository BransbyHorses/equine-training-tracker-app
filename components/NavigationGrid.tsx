import { Grid } from "@mui/material";

const NavigationGrid = (props: any) => {
	return (
		<Grid container rowSpacing={3} columnSpacing={2}>
			{props.children}
		</Grid>
	);
};
export default NavigationGrid;
