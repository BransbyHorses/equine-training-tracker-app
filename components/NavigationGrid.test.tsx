import renderer from "react-test-renderer";
import "@testing-library/jest-dom";
import NavigationGrid from "./NavigationGrid";

describe("NavigationGrid", () => {
	it("renders", () => {
		const tree = renderer.create(<NavigationGrid />);
		expect(tree).toMatchSnapshot();
	});
});
