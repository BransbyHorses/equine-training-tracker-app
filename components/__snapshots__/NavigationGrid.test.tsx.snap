// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationGrid renders 1`] = `
<div
  className="MuiGrid-root MuiGrid-container css-y8o7sp-MuiGrid-root"
/>
`;
